centreon/packaging/scripts/centreon-common-postinstall.sh

```diff
@@ -44,3 +44,7 @@ case "$action" in
     startCentreon
     ;;
 esac
+
+# MON-69138
+chown centreon:centreon /var/cache/centreon
+chmod 755 /var/cache/centreon
```

Comment:
you should create a function `fixSymfonyCacheRights` which you call on install and upgrade (like `fixCacheConfigRights`)